README.md

```diff
@@ -72,6 +66,19 @@ Append `HEAD:master` to the pit push if you are on a branch and want to push tha
 git push git@heroku.com:<app>.git HEAD:master
 ```
 
+Logs database
+-------------
+
+To run the migration commands for the logs database, set `RAILS_ENV=development_logs` as an environment variable.
+
```

Comment:
So, if I'm understanding this correctly,  it looks like this is something I'll need to account for here: https://github.com/travis-pro/travis-enterprise/blob/master/src/main/bin/te-db-migrate (add a second line that changes the `RAILS_ENV` to `development_logs`)?

Will these migrations still come with `travis-api` (or in the case of enterprise `travis-pro-api`) or will I need to package this application separately to run the migrations?